### DISCUSSION TITLE
求助，搭建服务器不会

### DISCUSSION THREAD

**Discussion by jian4310** [SeaHOH/GotoX#80] [一般]
我想请教一下这个怎么操作，想建一个自己的VPN，买了服务器不会用
![relnzoe5glpq wh uk2ig5](https://user-images.githubusercontent.com/39278250/40022537-e462e5f0-57fa-11e8-9656-0fb85e2b674b.png)

![u095c de6 d rvf 38wnomb](https://user-images.githubusercontent.com/39278250/40022447-9c67fb46-57fa-11e8-828c-87a72d2e5f99.png)

**Comment by SeaHOH** [comment]
暂时只支持谷歌，你可以考虑使用其它代理。
如：goflyway、gsnova 等
